searchlib/src/vespa/searchlib/attribute/reference_mappings.h

```diff
@@ -52,13 +55,24 @@ class ReferenceMappings
     void removeReverseMapping(const Reference &entry, uint32_t lid);
     void addReverseMapping(const Reference &entry, uint32_t lid);
 
+    // Maintain size of mapping from lid to referenced lid
+    void onAddDocs(uint32_t limit);
```

Comment:
Should we consolidate naming here and rename limit -> docIdLimit and numDocs -> docIdLimit?